inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -159,6 +159,23 @@ public void registerKnowledgeBase(KnowledgeBase kb, RepositoryImplConfig cfg)
         repoManager.addRepositoryConfig(new RepositoryConfig(repositoryId, cfg));
         entityManager.persist(kb);
     }
+    
+    @Override
+    public void defineBaseProperties(KnowledgeBase kb, boolean kbUpdateFlag) 
+    {
+        // KB will initialize base properties with base IRI schema properties defined by user
+        if (kb.getType() == RepositoryType.LOCAL) {
+            KBProperty property = new KBProperty(kb.getSubclassIri().getLocalName());
```

Comment:
I'd suggest to introduce a constructor for `KBProperty` with two arguments: identifier and name - and then to use that here.